drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -339,49 +339,51 @@ def data_collection_features
     end.compact
   end
 
+  # Occurrence Point Forms that are enabled for this Enrollment.
+  # Returns array of OpenStructs, which are resolved by the HmisSchema::OccurrencePointForm GQL type.
   def occurrence_point_forms
```

Comment:
This is fine; it's consistent with how the other models. But the Enrollment class is getting kind of big (550 lines). I wonder if this might be an opportunity to extract this into some kind of single-purpose service class
```ruby
class Enrollment 
  #...
  def occurrence_point_forms
    OccurrencePointFormCollection.new(self).all
  end
end

class OccurrencePointFormCollection
  def initialize(enrollment)
    @enrollment = enrollment
  end

  def all =  active_forms + legacy_forms

  def active_forms
    definitions = Hmis::Form::Definition
      .with_role(:OCCURRENCE_POINT)
      .published_or_retired
      .latest_versions
    # definitions.map ...
  end

  def legacy_forms
    #...
  end
end